src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -204,9 +204,7 @@
       RestoreConfigFile property and uses NuGet.config file from repo's root which has online feeds.
     -->
     <NoTargetsSdkOverride Include="Microsoft.Build.NoTargets" Group="NOTARGETS" Version="$(NOTARGETS_BOOTSTRAP_VERSION)" />
-    <TraversalSdkOverride Include="Microsoft.Build.Traversal" Group="TRAVERSAL" Version="$(TRAVERSAL_BOOTSTRAP_VERSION)" />
     <SourceBuiltSdkOverride Include="@(NoTargetsSdkOverride)" Condition="'$(DotNetBuildSourceOnly)' == 'true' and '$(RepositoryName)' != 'source-build-reference-packages'" />
```

Comment:
Asking to better understand this change. Why did you remove the `Microsoft.Build.Traversal` but not `Microsoft.Build.NoTargets` items?

Reply:
Honestly I just missed it when moving the various pieces around. I'll clean it up.